scholly/base/src/index.js

```diff
@@ -9,4 +10,9 @@ import 'bootstrap/dist/css/bootstrap-reboot.css';
 import App from "./components/App";
 
 
-ReactDOM.render(<App />, document.getElementById('root'));
\ No newline at end of file
+ReactDOM.render(
+    <HashRouter>
+        <App />
+    </HashRouter>,
```

Comment:
WHat's `HashRouter` used for?

Reply:
Good catch, I wanted to explain it on the PR but I forgot. 
<HashRouter>is replacing the <BrowserRouter> since GitHub Pages does not support browser history. If you try running the program by using the <BrowserRouter> tag, the page gets 404 error when refreshing the page/ navigating to other pages. 

Using HashRouter solved the problem, but we'll have a hash symbol on deployed website URL e.g. https://pmyip3.github.io/#/team. Not sure if there are any other ways to solve the problem.  

Read more on https://www.freecodecamp.org/news/deploy-a-react-app-to-github-pages/
https://stackoverflow.com/questions/51974369/what-is-the-difference-between-hashrouter-and-browserrouter-in-react

Reply:
@pmyip3 good catch. Can we document this somewhere, e.g. on the team drive? It seems to be an interesting problem that may be worth documenting.